apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -201,18 +202,17 @@ export class DsbMessagePoolingService implements OnModuleInit {
         this.logger.warn(`[WS][sendMessagesToSubscribers][sendAckBy] ${e}`);
         return [];
       });
-      const idsNotAck: string[] = messages.filter(e => !successAckMessageIds.includes(e.id)).map(e => e.id);
-
-      let saveAcks: AcksEntity[] = [];
-      idsNotAck.forEach(messageId => {
-        const ack = new AcksEntity();
-        ack.clientId = clientId;
-        ack.messageId = messageId;
-        saveAcks.push(ack);
+      const idsNotAck: AcksEntity[] = messages.filter(e => !successAckMessageIds.includes(e.id)).map(e => {
+        return {
+          clientId: clientId,
+          messageId: e.id,
+          mbTimestamp: moment(e.timestampNanos / (1000 * 1000)).utc().toDate(),
+          createdDate: moment().utc().toDate()
```

Comment:
`createdDate` is automatically filled by `typeorm`, no need to do it

Reply:
need to remove ! for createdDate